docs/docs/themes/getting-started.md

```diff
@@ -1,35 +1,56 @@
 ---
-title: Getting Started
+title: Using a Gatsby theme
 ---
 
-Scaffolding out a project that uses themes requires you to use a starter that's implemented with theming.
+To get started using themes, we'll use a starter that installs the official gatsby blog theme.
+
+A **regular Gatsby starter** creates a new Gatsby site that is preconfigured for a particular use case. The resulting site effectively forks off the starter — after it’s created, the site maintains no connection to the starter.
+
+A **Gatsby theme starter** creates a new site that installs one or more Gatsby themes. When a starter installs a theme, it maintains the connection to that theme as a standalone npm package.
 
 ```shell
 gatsby new my-blog https://github.com/gatsbyjs/gatsby-starter-blog-theme
 ```
 
-When you use a starter that's built with a theme, you will see that you're initially presented with a lighter weight `gatsby-config.js`. The goal with most themes is to allow you to get up and running with something like a blog in seconds. Now you can focus on writing and gradually introduce changes and customizations only if you want to.
+## What does a theme starter do?
+
+The starter for the official gatsby blog theme does the following:
 
-For example, a theme-based starter for a blog might do the following:
+### 1. The starter installs the theme and configures it
 
-## 1. Install the Theme and Configure It
+When you use a starter that's built with a theme, you will see that you're initially presented with a lighter weight `gatsby-config.js`.
```

Comment:
```suggestion
When you use a starter that's built with a theme, you will often see that you're initially presented with a lighter weight `gatsby-config.js`. Themes start doing their magic when installed via the `plugins` array:
```

I think it's important to not set the expectation that *every* theme starter would be lighter weight. Also, we should probably point out the plugins array as notable (feel free to reword it)